prog/main.go

```diff
@@ -115,12 +117,14 @@ func main() {
 		flags         = flags{}
 		mode          string
 		debug         bool
+		weaveEnabled  bool
 		weaveHostname string
 	)
 
 	// Flags that apply to both probe and app
 	flag.StringVar(&mode, "mode", "help", "For internal use.")
 	flag.BoolVar(&debug, "debug", false, "Force debug logging.")
+	flag.BoolVar(&weaveEnabled, "weave", true, "Enable Weave Net integrations.")
```

Comment:
Okay decide we don't want to turn it off by default, and @peterbourgon also doesn't like -noweave, so we'll leave as is.